libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteArrow.java

```diff
@@ -240,7 +240,9 @@ private void initializeArrowHead(int sourceMaxZoom, float sourceTolerance) {
   private void addArrowHeadIcon() {
     int headResId = R.drawable.mapbox_ic_arrow_head;
     Drawable arrowHead = AppCompatResources.getDrawable(mapView.getContext(), headResId);
-    if (arrowHead == null) {
+    if (arrowHead == null
+            || arrowHead.getIntrinsicHeight() < 0
+            || arrowHead.getIntrinsicWidth() < 0) {
```

Comment:
Can we add a Unit test covering the new logic added here?

Reply:
sounds good :trollface: